game-core/src/main/java/games/strategy/engine/random/MartiDiceRoller.java

```diff
@@ -43,7 +43,7 @@
   // Matches a comma separated list of integers like this:
   // your dice are: 1,2,3 <p>
```

Comment:
This comment probably could use elaboration and give a number more examples of what we are supposed to match.

Reply:
bump, this comment is out of date and/or inadequate since the regex matches far more than just the above example. Expecting single digits is part of the problem of the original regex, this comment is misleading (arguably greatly contributed to this bug)